data/Debian.yaml

```
@@ -0,0 +1,14 @@
+---
+rundeck::override_dir: '/etc/default'
+
+rundeck::repo_config:
+  'apt::source':
+    'rundeck':
+      location: 'https://packagecloud.io/pagerduty/rundeck/any'
+      release: 'any'
+      repos: 'main'
+      comment: 'Official repository for Rundeck'
+      key:
+        id: '0DDD2FA79B15D736ECEA32B89B5206167C5C34C0'
+        source: 'https://packagecloud.io/pagerduty/rundeck/gpgkey'
+        server: 'keyserver.ubuntu.com'
```

Use modern APT keyring, the keyserver way is deprecated.

Also remove the `apt::source` level so that you can just iterate over `rundeck::repo_config`. Same for `RedHat.yaml`.

```suggestion
rundeck::repo_config:
  rundeck:
    location: https://packagecloud.io/pagerduty/rundeck/any
    release: any
    repos: main
    comment: Official repository for Rundeck
    key:
      name: rundeck.asc
      source: https://packagecloud.io/pagerduty/rundeck/gpgkey
```